quinn-proto/src/connection/mod.rs

```diff
@@ -1470,15 +1470,20 @@ impl Connection {
                 lost_packets,
                 size_of_lost_packets
             );
+
             for packet in &lost_packets {
                 let info = self.spaces[pn_space].sent_packets.remove(packet).unwrap(); // safe: lost_packets is populated just above
                 self.remove_in_flight(pn_space, &info);
                 for frame in info.stream_frames {
                     self.streams.retransmit(frame);
                 }
                 self.spaces[pn_space].pending |= info.retransmits;
-                self.path.mtud.on_non_probe_lost(now, *packet, info.size);
+                self.path.mtud.on_non_probe_lost(*packet, info.size);
             }
+
+            let black_hole_detected = self.path.mtud.black_hole_detected(now);
```

Comment:
Style nit: I think this would be easier to understand if it didn't try to be fancy with the `bool` to `u64` conversion, instead writing this as an `if`-block.

Reply:
I agree, just updated it